backend/svirt.pm

```diff
@@ -167,25 +170,33 @@ sub save_snapshot {
 
 sub load_snapshot {
     my ($self, $args) = @_;
-    my $snapname = $args->{name};
-    my $vmname   = $self->console('svirt')->name;
-    my $rsp;
+    my $snapname                   = $args->{name};
+    my $vmname                     = $self->console('svirt')->name;
     my $post_load_snapshot_command = '';
+    my $vm_state;
+    my $rsp;
     if (check_var('VIRSH_VMM_FAMILY', 'hyperv')) {
         my $ps = 'powershell -Command';
+        $self->run_ssh_cmd("$ps (Get-VMCheckpoint -VMName $vmname).Name");
+        (undef, $vm_state) = $self->run_ssh_cmd("$ps (Get-VM -VMName $vmname).State", wantarray => 1);
+        $vm_state =~ s/[\r\n]+$//;
```

Comment:
Maybe using `Mojo::Util::trim()` https://docs.mojolicious.org/Mojo/Util#trim makes it easy.

Reply:
```
[0m[37m[2021-05-27T12:45:41.662 CEST] [debug] [run_ssh_cmd(powershell -Command (Get-VM -VMName openQA-SUT-8).State)] stdout:
  Running
  
[0m[37m[2021-05-27T12:45:41.663 CEST] [debug] [run_ssh_cmd(powershell -Command (Get-VM -VMName openQA-SUT-8).State)] exit-code: 0
[0m[37m[2021-05-27T12:45:41.663 CEST] [debug] Restoring RDP connection, attempts: 1, VM state: Running
  
[0m[2021-05-27T12:45:41.664 CEST] [debug] <<< backend::baseclass::run_ssh_cmd(cmd="pgrep --full --list-full xfreerdp.*\$(cat xfreerdp_openQA-SUT-8_stop.bkp)", password="SECRET", keep_open=1, wantarray=0, username="root", hostname="openqaw5-xen-1.qa.suse.de")
  ```
  
Trim does not seem to work well when it comes to Windows output.

Reply:
sorry then for the suggestion, I didn't expect that. But I'm still confused.
As this is the implementation https://docs.mojolicious.org/Mojo/Util.txt
```
sub trim {
  my $str = shift;
  $str =~ s/^\s+//;
  $str =~ s/\s+$//;
  return $str;
}
```
And I thought `\r` and `\n` are in the `\s` class. https://perldoc.perl.org/perlrecharclass#Whitespace

Reply:
My fault! :facepalm: Works well, too. Thanks!